app/utils/RepaymentAmountSummaryAnswersHelper.scala

```diff
@@ -29,8 +29,8 @@ class RepaymentAmountSummaryAnswersHelper(userAnswers: UserAnswers)(implicit mes
     def format2d = "%.2f".format(s)
   }
 
-  def displayDuty(index: String, amount: Double, dutyType: String): Option[AnswerRow] =
-    getAnswerRow(index, amount, dutyType)
+  def displayDuty(index: String, amount: Double, dutyType: String): AnswerRow =
+    getAnswerRow(index, amount, dutyType).getOrElse(AnswerRow(Html(""), Html("")))
```

Comment:
CLARIFICATION: Do these fixes in this file stop the Repayment Page throwing an exception?  If so, I have a bug report on the backlog that can be tied to this story and tested alongside.

Reply:
They do.   I remember you mentioning a repayment summary page bug yesterday.  I (re)discovered this when manually navigating to CYA with no answers in cache and trying to "change" the repayment amount.  In reality, I don't think there should be such an extreme example of missing answers.